env0/resource_cloud_credentials_project_assignment_test.go

```diff
@@ -0,0 +1,148 @@
+package env0
+
+import (
+	"errors"
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"regexp"
+	"testing"
+)
+
+func TestUnitResourceCloudCredentialsProjectAssignmentResource_Create(t *testing.T) {
+	resourceType := "env0_cloud_credentials_project_assignment"
+	resourceName := "test"
+	accessor := resourceAccessor(resourceType, resourceName)
+	assignment := client.CloudCredentialsProjectAssignment{
+		CredentialId: "cred-it",
```

Comment:
I think you can define 
`{		CredentialId: "cred-it",		ProjectId:    "proj-it",	}` and reuse it in few place in tests

Reply:
62b2c69a took @avnerenv0 method for reuse stuff